src/provider/config-provider.js

```diff
@@ -1,20 +1,22 @@
 import { writable } from "svelte/store";
+import allLocale from "./locale";
 
 export const CONFIG_KEY = "$config";
 export function configProvider({
   getPopupContainer,
   prefixCls = "ant",
   csp,
-  locale,
+  locale = "en_US",
```

Comment:
I would keep this an object since we might not support every language and we want the user to be able to specify the local.

Reply:
Ah okay, so if the user wants to use a different one, they'd import it in directly and pass it to the ConfigProvider. Makes sense 👍